frontend_lib/src/TracimTableLib/Columns/content/contentTypeColumn.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { createColumnHelper } from '@tanstack/react-table'
+import ContentType from '../../../component/ContentType/ContentType.jsx'
+
+const contentTypeColumn = (settings, contentType, t) => {
+  const columnHelper = createColumnHelper()
+  return columnHelper.accessor(row => {
+    const contentTypeInfo = contentType.find(info => info.slug === row.originalType) || { label: '' }
+    return t(contentTypeInfo.label)
+  }, {
+    header: () => (
+      <span>{settings.header}</span>
+    ),
+    id: 'type',
```

Comment:
One improvement (that doesn't necessarily need to be done in this PR but might be good to not take too long to do) is to put all these IDs in a constant, an object, so we have all of them in the same place and when creating new column types we don't have the risk of using the same id twice.

Reply:
Indeed, I will see if I implement it in this PR !

Reply:
Now that the id are defined in each tables, the use of constants is not necessary.